src/test/java/uk/gov/homeoffice/digital/sas/timecard/testutils/TestConstants.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.homeoffice.digital.sas.timecard.testutils;
+
+public class TestConstants {
```

Comment:
Instead of creating TestConstants, why not re-use Constants and substitute placeholders in the tests accordingly?

Reply:
Actioned

Reply:
you can do the same with these  2 remaining test constants : MESSAGE_FAILED_SENDING_TO_TOPIC and MESSAGE_SENT_TO_TOPIC_CALLISTO
After that, class TestConstants can be deleted altogether

Reply:
Thought I had done that. Actioned